pkgs/servers/monitoring/buildkite-agent-metrics/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, go, buildGoPackage, fetchFromGitHub }:
+
+let
+  goPackagePath = "github.com/buildkite/buildkite-agent-metrics";
```

Comment:
Can't see a reason not to just put this in the derivation

Reply:
Great point @alexarice - I was just rather blindly following the convention of related Go packages, such as Prometheus. I've corrected this in the most recent commit.